ui/main.glade

```diff
@@ -315,7 +315,7 @@
                       <object class="GtkCheckMenuItem" id="menuViewTwoPages">
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
-                        <property name="label" translatable="yes">_Two Pages</property>
```

Comment:
Same as above, this is the flag for all column views. Please name it accordingly.